packages/payment-integration-api/src/billing/is-billing-address-like.ts

```diff
@@ -0,0 +1,9 @@
+import { Address } from '../address';
+
+import BillingAddress from './billing-address';
+
+export default function isBillingAddressLike(address: Address): address is BillingAddress {
+    const billingAddress = address as BillingAddress;
```

Comment:
Can we avoid using as here?

Reply:
Serhii already asked about it, but looks like assertion is necessary here.

Reply:
We can check for properties in the object, that is much better than using assertion. Since using assertion is an override, you can call a `string as number` and it will be fine.

Reply:
Done